tokenomics-explainers/001-why-rework.md

```diff
@@ -82,7 +82,7 @@ The 10% borrowed-ETH minimum is something the protocol tries to impose upon the
 
 Consider a mature case with a 3% solo APY and 1.5% RPL value loss per year due to supply inflation. Using the same assumptions as earlier, this results in a **2.3% drop** compared to solo staking rewards. In such a case, there is no reason to make minipools if you can afford solo validators.
 
-This does present a solution as well – the minimum staked RPL can be reduced. Note that this would reduce the RPL to rETH TVL relationship per **Ideal 3**, which means that the cost to avoiding this brittleness is letting go of a relationship that can be known ahead of time.
```

Comment:
Makes it seem like we are suggesting a fix by lowering the requirement. Instead we are just highlighting a theoretical fix we aren't actually going to implement due to the effects it has on "Ideal 3"